data/json/vehicleparts/roof.json

```diff
@@ -0,0 +1,45 @@
+[
+  {
+    "id": "roof_wood",
+    "type": "vehicle_part",
+    "name": "wooden roof",
+    "item": "frame_wood",
+    "color": "brown",
+    "durability": 130,
+    "breaks_into": [ { "item": "splinter", "count": [ 7, 9 ] } ],
+    "mounts": [ { "name": "on_roof", "volume": 20, "weight": 1000 } ],
```

Comment:
`volume` is specified as I want to support multiple items in a location - eg. one large item on a roof or two smaller up to their total sum volume.